CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ _Sep 10, 2024_
 A big thanks to the 20 contributors who made this release possible. Here are the highlights ✨:
 
 - **Minor version bump**: This release includes a minor version bump as the `exports` field was added to the `@mui/icons-material` package to solve longstanding ESM compatibility issues (#43624). This change is backwards compatible with previously recommended setups. Bundlers which don't support the `exports` field shouldn't be affected. If you have any issues, please reach out by creating a GitHub issue.
```

Comment:
I think this could be enough. Perhaps we it could be rephrased a bit?
```suggestion
- **Minor version bump**: To solve longstanding ESM compatibility issues (#43624) we added package [exports](https://nodejs.org/api/packages.html#exports) to @mui/icons-material`. This change is backwards compatible with previously recommended setups. Bundlers that don't support the `exports` field won't be affected. If you discover any issues, please reach out by creating a GitHub issue.
```